src/test/scala/com/wix/mysql/EmbeddedMysqlTest.scala

```diff
@@ -16,7 +16,7 @@ class EmbeddedMysqlTest extends IntegrationTest {
     "start with default values" in {
       val config = aMysqldConfig(Version.v5_7_latest).build
 
-      mysqld = anEmbeddedMysql(config).start
+      val mysqld = withStop(anEmbeddedMysql(config).start)
```

Comment:
withStop -> start as there are too many variations of instance creation - simple, easy, less verbose